src/bitmessagekivy/baseclass/qrcode.py

```diff
@@ -0,0 +1,26 @@
+# pylint: disable=import-error, no-name-in-module, too-few-public-methods
+
+"""
+Genarate QRcode of saved addresses in addressbook.
+"""
+
+from kivy.uix.screenmanager import Screen
+from kivy.properties import StringProperty
+from kivy_garden.qrcode import QRCodeWidget
+from bitmessagekivy.baseclass.common import toast
+import state
+
+
+class ShowQRCode(Screen):
+    """ShowQRCode Screen class for kivy Ui"""
+    address = StringProperty()
+
+    def qrdisplay(self, instasnce, address):
+        """Method used for showing QR Code"""
+        self.ids.qr.clear_widgets()
+        state.kivyapp.set_toolbar_for_QrCode()
+        self.address = address
```

Comment:
why are we using `self.address`?

Reply:
Yes we can remove it, it is not using here.